apps/src/storage/dataBrowser/DataLibraryPane.jsx

```diff
@@ -26,18 +29,32 @@ const styles = {
 };
 
 class DataLibraryPane extends React.Component {
+  static propTypes = {
+    // from redux dispatch
+    onShowWarning: PropTypes.func.isRequired
+  };
+
+  onError = error => {
+    if (
+      typeof error === 'string' &&
+      error.includes('There is already a table with name')
+    ) {
+      this.props.onShowWarning(error);
+    }
+  };
+
   importTable = datasetInfo => {
     if (datasetInfo.current) {
       FirebaseStorage.addCurrentTableToProject(
         datasetInfo.name,
         () => console.log('success'),
```

Comment:
Is this console.log a long-term feature? If so, can you give it a more descriptive message?

Reply:
This is still under active development, so I want to keep the console.log around for now, but it won't be in the final feature.

Reply:
👍